pyquil/quilbase.py

```diff
@@ -2772,41 +2772,40 @@ def __new__(
         """Initialize a new calibration definition."""
         return super().__new__(
             cls,
-            name,
-            _convert_to_rs_expressions(parameters),
-            _convert_to_rs_qubits(qubits),
+            quil_rs.CalibrationIdentifier(
+                name,
+                _convert_to_rs_expressions(parameters),
+                _convert_to_rs_qubits(qubits),
+                modifiers or [],
+            ),
             _convert_to_rs_instructions(instrs),
-            modifiers or [],
         )
 
     @classmethod
     def _from_rs_calibration(cls, calibration: quil_rs.Calibration) -> "DefCalibration":
-        return super().__new__(
-            cls,
-            calibration.name,
-            calibration.parameters,
-            calibration.qubits,
-            calibration.instructions,
-            calibration.modifiers,
-        )
+        return super().__new__(cls, calibration.identifier, calibration.instructions)
 
     @property  # type: ignore[override]
     def parameters(self) -> Sequence[ParameterDesignator]:
         """The parameters of the calibration."""
-        return _convert_to_py_expressions(super().parameters)
+        return _convert_to_py_expressions(super().identifier.parameters)
```

Comment:
For some reason I can't get the type stubs to agree, looks like https://github.com/rigetti/quil-rs/issues/420 is resolved because the types are correct in https://github.com/rigetti/quil-rs/blob/main/quil-py/quil/instructions/__init__.pyi#L888-L897 but not when I install `quil-rs==0.13.2`... but, this should be the same thing and gets the test passing.